apps/examples/testcase/le_tc/kernel/tc_libc_string.c

```diff
@@ -396,12 +398,11 @@ static void tc_libc_string_strdup(void)
 static void tc_libc_string_strerror(void)
 {
 	char *dest_arr = NULL;
-	char src[BUFF_SIZE_12] = "Bad address";
 
 	/* EFAULT is defined as 14 which gives Bad address in strerror */
 	dest_arr = (char *)strerror(EFAULT);
 	TC_ASSERT_NEQ("strerror", dest_arr, NULL);
-	TC_ASSERT_EQ("strerror", strncmp(dest_arr, src, BUFF_SIZE_12), 0);
+	TC_ASSERT_EQ("strerror", strncmp(dest_arr, "Bad address", strlen("Bad address") + 1), 0);
```

Comment:
Instead of `strlen("Bad address") + 1`, we can use `sizeof("Bad address")`.
`sizeof` is better if the length can be translated at compile time.

Reply:
@junmin-kim I agree. But let's do that with next commit.
@hasw7569 Additionally let use `EFAULT_STR` instead of `"Bad address"`

Reply:
I agree too.
I will change that next commit.